### PR TITLE
Sync packages for WP 6.1 Beta 2

### PR DESCRIPTION
**Pull request by ockham**
Includes the latest round of Gutenberg bugfix PRs that were approved for inclusion during [yesterday’s triage session](https://wordpress.slack.com/archives/C02QB2JS7/p1664205112311519).

Needed for today's WP 6.1 Beta 2 release (scheduled in ~3 hours from now).

Trac ticket: https://core.trac.wordpress.org/ticket/56467

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/blocks/calendar.php [hunk 1]
```diff
@@ -40,11 +40,32 @@ function render_block_core_calendar( $attributes ) {
...
+	$custom_text_color          = _wp_array_get( $attributes, array( 'style', 'color', 'text' ), null );
+	$color_block_styles['text'] = $preset_text_color ? $preset_text_color : $custom_text_color;
+
+	// Background Color.
+	$preset_background_color          = array_key_exists( 'backgroundColor', $attributes ) ? "var:preset|color|{$attributes['backgroundColor']}" : null;
+	$custom_background_color          = _wp_array_get( $attributes, array( 'style', 'color', 'background' ), null );
+	$color_block_styles['background'] = $preset_background_color ? $preset_background_color : $custom_background_color;
+
+	// Generate color styles and classes.
+	$styles        = gutenberg_style_engine_get_styles( array( 'color' => $color_block_styles ), array( 'convert_vars_to_classnames' => true ) );
```
Comment (hellofromtonya) [inline src/wp-includes/blocks/calendar.php:56]: @ockham The prefix should be changed to `wp_` instead of `gutenberg_`.

Comment (ockham) [reply src/wp-includes/blocks/calendar.php:56]: Ugh, I missed that one. I'll fix it.

Comment (ockham) [reply src/wp-includes/blocks/calendar.php:56]: Sidenote, I don't quite understand how this ended up here. The relevant GB PR (https://github.com/WordPress/gutenberg/pull/42029) was only [merged](https://github.com/WordPress/gutenberg/commit/dc57581be214d5c2e6b77cfabf06dc75ffc374cf) after GB 14.1 was released, and we didn't cherry-pick it to the `wp/6.1` branch. In fact, the relevant code is absent from [that file in Gutenberg's `wp/6.1` branch](https://github.com/WordPress/gutenberg/blob/wp/6.1/packages/block-library/src/calendar/index.php) 🤔

Comment (ockham) [reply src/wp-includes/blocks/calendar.php:56]: Looks like the script picked up the wrong package version(s): [For `@wordpress/block-library`, I see `7.15.0`](https://github.com/WordPress/wordpress-develop/pull/3342/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R86).

But the [`wp-6.1` tag points to `7.14.3`](https://www.npmjs.com/package/@wordpress/block-library).

I think we had a similar issue before with https://github.com/WordPress/wordpress-develop/pull/3154, but I'm not 100% sure what @gziolo did to fix it...

**Review by ockham — COMMENTED** [review]
Please hold off reviewing this -- it appears that [the sync script picked up the wrong package versions](https://github.com/WordPress/wordpress-develop/pull/3342#discussion_r981309395).

File: src/wp-includes/blocks/calendar.php (hunk 1 above)
Comment (ockham) [reply src/wp-includes/blocks/calendar.php:56]: I think I forgot to specify the `dist-tag`. Note to future self:

```
npm run sync-gutenberg-packages -- --dist-tag=wp-6.1
```

Comment (ockham) [reply src/wp-includes/blocks/calendar.php:56]: (Re-syncing now.)

**Comment by ockham** [comment]
I've reverted the original commit and run `npm run sync-gutenberg-packages -- --dist-tag=wp-6.1` locally now. I've pushed the commits to this branch but don't see them yet here 😕 

FWIW, [GitHub Status](https://www.githubstatus.com/) is showing some issue with GH Actions (but no other GH components).

**Comment by ockham** [comment]
Closing. This PR has been flaky, and we've decided that we don't want to include https://github.com/WordPress/gutenberg/pull/43862, so we'll need to revert that on GB's `wp/6.1` branch and then re-publish npms.

**Comment by ockham** [comment]
New sync PR: https://github.com/WordPress/wordpress-develop/pull/3345

**Pull request closed without merging**